application/templates/deednotfound.html

```diff
@@ -2,6 +2,9 @@
 
 {% block content %}
 <div class="main" id="content">
+
+{{ super() }}
```

Comment:
1. What is super for?
2. Why are we calling super in some places and not in others?